tools/wemod_enhancer.py

```
@@ -133,19 +127,14 @@ def find_bundled_dll() -> Path | None:
 
-def build_dll() -> Path:
-    build = ROOT / "build" / "proxy"
-    args = ["cmake", "-S", str(ROOT), "-B", str(build), "-G", "Ninja", "-DCMAKE_BUILD_TYPE=Release"]
-    if os.name != "nt": args += ["-DCMAKE_TOOLCHAIN_FILE=" + str(ROOT / "cmake/toolchains/llvm_mingw.cmake"), "-DCMAKE_SYSTEM_PROCESSOR=x86_64"]
-    subprocess.run(args, check=True); subprocess.run(["cmake", "--build", str(build), "--config", "Release"], check=True)
-    dll = find_dll(build); pe_x64(dll); return dll
-
 def resolve_dll(explicit: Path | None) -> Path:
-    """Pick version.dll: explicit flag > bundled > build from source."""
+    """Pick version.dll: explicit flag > bundled next to this script."""
     if explicit: return explicit
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)